### PR TITLE
Chef 16 Fixes

### PR DESCRIPTION
**Pull request by zimmerry**

### CODE REVIEW

**Comment by zimmerry** [comment]
[Chef 16: yum-kernel-osuosl](https://trello.com/c/6EiUAYUM/1985-chef-16-yum-kernel-osuosl)

File: test/integration/install/install_spec.rb [hunk 1]
```diff
@@ -7,5 +7,5 @@
 end
 
 describe command('grubby --default-kernel') do
-  its('stdout') { should match(/4.19.*-osuosl/) }
+  its('stdout') { should match('5.4.*-osuosl') }
```
Comment (ramereth) [inline test/integration/install/install_spec.rb]: I'd prefer this.
```suggestion
  its('stdout') { should match/'5.4.*-osuosl/) }
```

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Please do a patch bump.

**Comment by zimmerry** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.1.2.  Have a nice day!

**Pull request merged**